### PR TITLE
Todoist api + unit tests finished

### PR DESCRIPTION
**Pull request by Ovakefali13**

### CODE REVIEW

File: services/todoAPI/test/test_service.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import unittest
+from .. import TodoistService, TodoistRemote, TodoistJSONRemote
+from ...preferences.pref_service import PrefService, PrefJSONRemote
```
Comment (JannisTriesToCode) [inline services/todoAPI/test/test_service.py]: This does probably work, but `from services.preferences import PrefService, PrefJSONRemote` will do same job, but does not rely on a relative path.

Comment (Ovakefali13) [reply services/todoAPI/test/test_service.py]: done

**Comment by mariushegele** [comment]
Seems like spoonacular has found its way into this merge request.

**Comment by Ovakefali13** [comment]
What do you mean wit PR ?

**Pull request merged**